vega_sim/scenario/constants.py

```diff
@@ -1,8 +1,9 @@
-from enum import Enum, auto
+from enum import Enum
 
 
 class Network(Enum):
-    NULLCHAIN = "nullchain"
-    STAGNET1 = "vegawallet-stagnet1"
-    STAGNET3 = "stagnet3"
-    FAIRGROUND = "fairground"
+    nullchain = "nullchain"
+    stagnet1 = "stagnet1"
+    stagnet2 = "stagnet2"
+    stagnet3 = "stagnet3"
+    fairground = "fairground"
```

Comment:
Why are these lowercase now?

Reply:
So the member refer to the directory name and the value refers to the name of the `.toml` config file in that directory. Before it assumed the directory name and config file name both matched the members value.

This is a workaround for the cases where the network file is prefixed with `vegawallet` whilst the directory isn't. Have just realised `stagnet2` is wrong. Will pushed a fix now.

Reply:
I would say these should ideally stay as capitals since they are enum constants. If we need to map to something else then an additional mapping dictionary would be best.

Reply:
Sounds good, have reverted back to capitalised Enum members 👍